api/services/ocean/ecs/paths/right-sizing-recommend.yaml

```diff
@@ -1,5 +1,5 @@
 get:
```

Comment:
@nogakinor1 I think this needs to be a POST call, according to the new instructions.
please check

Reply:
@sagishoshan ecs is not relevant to the change, so I didn't change it. what do you think?

Reply:
I think we need to check the actual routes to see what is working, and then have those calls aligned

Reply:
@sagishoshan I checked with containers team, it should remain get